web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -749,6 +753,210 @@ function SubscriptionUpsellPrompt({
   )
 }
 
+function KodiakSaveButton({ state }: { state: WebData<unknown> }) {
+  const loading = state.status === "loading"
+  return (
+    <>
+      <Button variant="dark" size="sm" disabled={loading} type="submit">
+        {loading ? "Loading..." : "Save"}
+      </Button>
+      {state.status === "failure" && (
+        <Form.Text className="text-danger">Save failed</Form.Text>
+      )}
+    </>
+  )
+}
+
+function BillingEmailForm({
+  defaultValue,
+  className,
+}: {
+  readonly defaultValue: string
+  readonly className?: string
+}) {
+  const [billingEmail, setBillingEmail] = React.useState(defaultValue)
+  const [apiState, updateStripeCustomerInfo] = useTeamApiMutation(
+    Current.api.updateStripeCustomerInfo,
+  )
+  function handleSubmit(e: React.ChangeEvent<HTMLFormElement>) {
+    e.preventDefault()
+    updateStripeCustomerInfo({ email: billingEmail })
+  }
+  return (
+    <Card className={className}>
+      <Card.Body>
+        <Card.Title>Billing Email</Card.Title>
+        <Form onSubmit={handleSubmit}>
+          <Form.Group>
+            <Form.Control
+              type="email"
+              required
+              value={billingEmail}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setBillingEmail(e.target.value)
+              }
+            />
+            <Form.Text className="text-muted">
+              Required. Address to send billing receipts.
```

Comment:
I think this required should be made bold, so it sticks out a bit more

Reply:
<img width="340" alt="image" src="https://user-images.githubusercontent.com/1929960/84602676-2f77f580-ae57-11ea-8356-70283124e36e.png">